test/suite

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#
+# Run like:
+# 
+#   ./test/suite
+#
+# To run individual tests, run the test file:
+#
+#   ./test/variable/variable.test.functions
+#
+# Use `-v` to see stderr on failing tests.  Use the test name to run just one test.
+#
+#   ./test/variable/variable.test.functions -v test_core_variable_startsWith
+#   
+
+find . -type f -name '*.test.*' |
+grep -v -E '(helper|suite)' |
+xargs ts "$@"
```

Comment:
This script should change to use the _program wrappers - see the overdrive tutorial - to make it robust on all platforms and helpful, too.